pkg/plugins/golang/v2/webhook.go

```diff
@@ -101,9 +101,11 @@ func (p *createWebhookSubcommand) InjectResource(res *resource.Resource) error {
 			return fmt.Errorf("%s create webhook requires a previously created API ", p.commandName)
 		}
 	} else {
-		if r, err := p.config.GetResource(p.resource.GVK); err != nil {
+		loadedResource, err := p.config.GetResource(p.resource.GVK)
+		if err != nil || !loadedResource.HasAPI() {
```

Comment:
I think that this condition now fails for `PROJECT` version 2. `go/v2` needs to support both `PROJECT` versions 2 and 3.

Reply:
It supports both. Check the code before `else`.

Reply:
Oh right that is checked before.

Why not just add the `|| !loadedResource.HasAPI()` to the first if condition?

Reply:
No reason. I thought new conditions can be added without extra else blocks that way. I can go with your suggestison.